### PR TITLE
Support for curly brackets

### PR DESCRIPTION
**Pull request by slavaschmidt**
Adding support for curly brackets as discussed. Made it explicit for eventual future use

### CODE REVIEW

File: src/main/scala/de/zalando/beard/ast/Nodes.scala [hunk 1]
```diff
@@ -49,6 +49,8 @@ trait HasText {
 
 case class Text(text: String) extends Statement with HasText
 
+case class CurlyBracket(text: String) extends Statement with HasText
```
Comment (danpersa) [inline src/main/scala/de/zalando/beard/ast/Nodes.scala]: I'm not sure we need this... shouldn't we transform this into a text?

File: src/main/antlr4/de/zalando/beard/BeardParser.g4 [hunk 2]
```diff
@@ -178,4 +179,9 @@ locals [NewLine result]
 text
 locals [Text result]
     : TEXT
```
Comment (danpersa) [inline src/main/antlr4/de/zalando/beard/BeardParser.g4:180]: I would put here

```
TEXT | CURLY_BRACKET
```

Comment (slavaschmidt) [inline src/main/antlr4/de/zalando/beard/BeardParser.g4:180]: May be then just make them allowed by removing from the line 9 of BeardLexer.g4 converting it into 
`TEXT    : ~(' ' | '\t' | '\n' | '\r')+;`
?
No other changes will be needed in this case.

Comment (danpersa) [inline src/main/antlr4/de/zalando/beard/BeardParser.g4:180]: I'll check if it works tomorrow ;)

Comment (danpersa) [inline src/main/antlr4/de/zalando/beard/BeardParser.g4:180]: this breaks lots of tests... :(

**Pull request merged**